frontend/src/editor/transcription_editor.tsx

```diff
@@ -40,7 +40,10 @@ function renderElement({ element, children, attributes }: RenderElementProps): J
           <div
             contentEditable={false}
             className="w-16 mr-2 -ml-20 hidden 2xl:block text-slate-500 dark:text-neutral-400 font-mono"
-            onClick={() => window.dispatchEvent(new TextClickEvent(startAtom))}
+            onClick={useCallback(
```

Comment:
how is this better to just passing the callback directly?

Reply:
if the component to render is cheap, this is not called for (see https://dmitripavlutin.com/react-usecallback/)